server/shared/src/main/scala/org/http4s/server/middleware/authentication/DigestAuth.scala

```diff
@@ -19,29 +19,63 @@ package server
 package middleware
 package authentication
 
+import cats.Functor
 import cats.Monad
 import cats.data.Kleisli
 import cats.data.NonEmptyList
 import cats.effect.Async
 import cats.effect.Sync
 import cats.syntax.all._
 import org.http4s.crypto.Hash
-import org.http4s.crypto.unsafe.SecureRandom
 import org.http4s.headers._
 
-import java.math.BigInteger
-import java.util.Date
 import scala.concurrent.duration._
 
 /** Provides Digest Authentication from RFC 2617.
   */
 object DigestAuth {
 
+  @deprecated(
+    "AuthenticationStore is going away, in favor of explicit subclasses of AuthStore. PlainTextAuthStore maintains the previous, insecure behaviour, whereas Md5HashedAuthStore is the new advised implementation going forward.",
+    "0.22.13",
```

Comment:
Keeping deprecated versions consistent within `0.22.`, `0.23`, `1.0` it's not a cakewalk. It seems like there should be `0.23`. Though, from my way of thinking, it's ok to set `1.0`.

Reply:
I used to just leave them, but it assumes too much familiarity with the linear order of releases. I think this one was a miss in the merge from 0.22.

Reply:
Feels like using dates instead of versions would save us a lot of trouble, but anyway 😆

Reply:
Commit date?  Merge date?  Release date?  Date it hits that release series?  The first is useless, and everything else is still some sort of chore.